src/utils/convertMarkdown.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { Fragment } from "react";
+
+import ReactMarkdown from "react-markdown";
+import glm from "remark-gfm";
+import emoji from "remark-emoji";
+import externalLinks from "remark-external-links";
+import sanitize from "rehype-sanitize";
+
+export const convertMarkdown = (
+  text: string | undefined,
```

Comment:
This is the shorthand way to write this, and matches our patterns.

```suggestion
  text?: string,
```